ts/components/ui/ListItemTransaction.tsx

```diff
@@ -0,0 +1,194 @@
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
+import * as React from "react";
+import { ImageURISource, StyleSheet, View } from "react-native";
+import Placeholder from "rn-placeholder";
+import I18n from "../../i18n";
+import { WithTestID } from "../../types/WithTestID";
+import { IOBadge } from "../core/IOBadge";
+import { Icon } from "../core/icons";
+import { IOLogoPaymentType, LogoPayment } from "../core/logos";
+import { VSpacer } from "../core/spacer/Spacer";
+import { LabelSmall } from "../core/typography/LabelSmall";
+import { NewH6 } from "../core/typography/NewH6";
+import { useIOTheme } from "../core/variables/IOColors";
+import {
+  IOListItemStyles,
+  IOListItemVisualParams,
+  IOStyles
+} from "../core/variables/IOStyles";
+import Avatar from "./Avatar";
+import {
+  PressableBaseProps,
+  PressableListItemBase
+} from "./utils/baseListItems";
+
+type LogoNameOrUri = IOLogoPaymentType | ImageURISource;
+export type ListItemTransaction = WithTestID<
+  PressableBaseProps & {
+    hasChevronRight?: boolean;
+    isLoading?: boolean;
+    leftPaymentLogoOrUrl?: LogoNameOrUri;
```

Comment:
There is only one logo (payment method or organization logo), so we probably don't need to specify the position. By the way, we could use logical properties (`start` and `end`) instead of relying on the classic `left` or `right` attributes.
```suggestion
    paymentLogoOrUrl?: LogoNameOrUri;
```